documentation/content/Contributing/automated-release.md

```diff
@@ -1,42 +1,93 @@
 
-<!-- yaspeller ignore:start -->
+    # [3.2.0](https://github.com/Pelican-Elegant/elegant/compare/V3.1.0...V3.2.0) (2019-07-30)
 
-| Commit Message                                                                                                                                                                             | Release Type           |
-| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------- |
-| fix(pencil): stop graphite breaking when too much pressure applied                                                                                                                         | Patch Release          |
-| feat(pencil): add 'graphiteWidth' option                                                                                                                                                   | Minor Feature Release  |
-| perf(pencil): remove graphiteWidth option<br><br>BREAKING CHANGE: The graphiteWidth option has been removed.<br>The default graphite width of 10mm is always used for performance reasons. | Major Breaking Release |
+    ### Bug Fixes
 
-<!-- yaspeller ignore:end -->
+    * **freelists:** open FreeLists subscription form in a new tab ([f81657c](https://github.com/Pelican-Elegant/elegant/commit/f81657c))
+    * **freelists:** replace deprecated subscription form with button ([9bfe3c1](https://github.com/Pelican-Elegant/elegant/commit/9bfe3c1)), closes [#412](https://github.com/Pelican-Elegant/elegant/issues/412)
+    * **freelists:** rm unused include ([27f0831](https://github.com/Pelican-Elegant/elegant/commit/27f0831))
 
-## Test Release Process Locally
+    ### Features
 
-To test semantic-release locally,
+    * **comments:** reduce transition duration from 500 to 200 ([b86e13d](https://github.com/Pelican-Elegant/elegant/commit/b86e13d))
+    * **favicon:** add 180x180 dimension shortcut icon support ([dd2ed24](https://github.com/Pelican-Elegant/elegant/commit/dd2ed24))
+    * **filter:** add black list, white list feature for Disqus ([4887aec](https://github.com/Pelican-Elegant/elegant/commit/4887aec))
+    * **filter:** add black list, white list feature for FreeLists ([2407cc8](https://github.com/Pelican-Elegant/elegant/commit/2407cc8))
+    * **filter:** add black list, white list feature for Mailchimp ([b96122d](https://github.com/Pelican-Elegant/elegant/commit/b96122d))
+```
 
-1. Install yarn
-1. Run `yarn semantic-release`
+In the main body of the output, there are links with associated text which are 7 digit
+hexadecimal numbers. Each of these links is to a specific commit that was part of the
+release. If you follow the link, you can verify that the correct text is being used for the
+release notes.
 
-It will run the all the checks and steps in dry run mode.
+Additionally, there are links that are associated with text that starts with an octothorpe
```

Comment:
Why go into so much detail? Why not just use `#` symbol?

Reply:
pet peeve. I hate it when some people call that symbol hash, or hashtag, or cross, or....

Fixed.